src/helpers/organizedStocksInGroups.tsx

```diff
@@ -0,0 +1,52 @@
+import getStoredDataFromStorage from "./getStoredDataFromStorage";
+import { getWeekNumber } from "./getWeekNumber";
+import { StocksData } from "./stockApi";
+
+export default function organizeStocksInGroups(
+  groupBy: string,
```

Comment:
This could be an enum, to ensure only valid options are being passed. E.g. right now I could pass "year" or "Month" or "months" and it would return weeks.